deps/wazuh_testing/wazuh_testing/scripts/wazuh_statistics.py

```diff
@@ -26,13 +26,20 @@ def get_script_arguments():
                                      formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument('-t', '--target', dest='target_list', required=True, type=str, nargs='+', action='store',
                         help='Type the statistics target to collect separated by whitespace. '
-                             'Targets: agent, logcollector, remote and analysis.')
+                             'Targets: agent, logcollector, remote, analysis and wazuhdb')
     parser.add_argument('-s', '--sleep', dest='sleep_time', type=float, default=5, action='store',
                         help='Type the time in seconds between each entry.')
     parser.add_argument('-d', '--debug', dest='debug', action='store_true', default=False,
                         help='Enable debug level logging.')
     parser.add_argument('--store', dest='store_path', action='store', default=gettempdir(),
                         help=f"Path to store the CSVs with the data. Default {gettempdir()}.")
+    parser.add_argument('-u', '--use_state_file', action='store_true', default=False,
+                        help="Determine if the state files should be used to collect the for analysisd and remoted."
+                             "Use with remoted and analysis to get data from state files. Default False.")
+    parser.add_argument('-i', '--ip', dest='ip', action='store', default='localhost',
+                        help=f"IP for the API. Default localhost.")
+    parser.add_argument('-p', '--port', dest='port', action='store', default='55000',
+                        help=f"port for the API. Default localhost.")
```

Comment:
```suggestion 
    parser.add_argument('-u', '--use_state_file', action='store_true', default=False,
                        help="Use state files for analysis and remote operations."
                             "When used with 'remote' and 'analysis', data will be collected from state files; otherwise, the API will be used. Default False.")
    parser.add_argument('-i', '--ip', dest='ip', action='store', default='localhost',
                        help=f"Specify the IP address for the API. Default is 'localhost'.")
    parser.add_argument('-p', '--port', dest='port', action='store', default='55000',
                        help=f"Specify the port for the API. Default is '55000'.")
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/4970/commits/a9192a52bea729bb7a79e26a3009572eefd8e666